tests/test_configs/mpi_multicpu.yaml

```diff
@@ -0,0 +1,22 @@
+compute_environment: LOCAL_MACHINE
```

Comment:
Let's tag a version name with this filename please (we can specify mpi as well)

E.g. 0_28_0_mpi.yaml